frontend/src/tests/components/Commons/ProfitsTable.test.js

```diff
@@ -25,4 +27,32 @@ describe("ProfitsTable tests", () => {
         });
 
     });
-});
\ No newline at end of file
+test("shows expected headers and fields for ProfitsTable", () => {
+  render(<ProfitsTable profits={profitsFixtures.threeProfits} />);
+
+  const headers = ["Profit", "Date", "Health", "Cows"];
+  const testId = "ProfitsTable";
+  
+
+  headers.forEach((header) => {
+    expect(screen.getByText(header)).toBeInTheDocument();
+  });
+
+  const firstProfit = profitsFixtures.threeProfits[0];
+  expect(screen.getByTestId(`${testId}-cell-row-0-col-Profit`)).toHaveTextContent(
+    `$${firstProfit.amount.toFixed(2)}`
+  );
+
+  expect(screen.getByTestId(`${testId}-cell-row-0-col-date`)).toBeInTheDocument(); // Date is empty as per the current rendering
```

Comment:
Remove this code if it doesn't do anything for now

Reply:
I think it check the date field, the comment is just when I figure the code out and forget to delete it. I will delete it and commit again

Reply:
Please update the branch to main as well

Reply:
already update to main